bigframes/series.py

```diff
@@ -1524,6 +1525,49 @@ def sample(
             self._block._split(ns=ns, fracs=fracs, random_state=random_state)[0]
         )
 
+    def hist(
```

Comment:
Can we expose this as `Series.plot.hist`, instead? https://pandas.pydata.org/pandas-docs/version/2.1/reference/api/pandas.Series.plot.hist.html

I'm not 100% sure why pandas provides Series.plot.hist and Series.hist https://pandas.pydata.org/pandas-docs/version/2.1/reference/api/pandas.Series.hist.html but  I think Series.plot.hist is the newer method and will make our docs organization easier.

Reply:
That was what I confused too! Thanks for your suggestions!

Reply:
Updated it into `Series.plot.hist`. Please review it again!